fe/fe-core/src/main/java/org/apache/doris/httpv2/rest/TableQueryPlanAction.java

```diff
@@ -163,65 +170,78 @@ public Object query_plan(
      */
     private void handleQuery(ConnectContext context, String requestDb, String requestTable, String sql,
             Map<String, Object> result) throws DorisHttpException {
-        // use SE to resolve sql
-        StmtExecutor stmtExecutor = new StmtExecutor(context, new OriginStatement(sql, 0), false);
+        List<StatementBase> stmts = null;
         try {
-            TQueryOptions tQueryOptions = context.getSessionVariable().toThrift();
-            // Conduct Planner create SingleNodePlan#createPlanFragments
-            tQueryOptions.num_nodes = 1;
```

Comment:
this is used to ensure generate one fragment plan. and will be check fragment num later. so in nereids, we should set enable_parallel_result_sink = true